pystare/exceptions.py,sha256=S15072jzWxWNz7DzG3WvMXYdgFAux-lq191nsL1JZ50,147
pystare/_core.cpython-38-x86_64-linux-gnu.so,sha256=RtYe4u7zm1TFAxqCTeXufuiLX8gDH0B-GcUW3M7Iwy8,14449176
pystare/spatial.py,sha256=06HoVv6-vzR9pfj4ubSCpJyNGKV7ua2BVcccYPzp4gU,30881
pystare/__init__.py,sha256=22vZaFiPFPik8Wixe748LF7O62fj5tpQHu76YxgeKa0,135
pystare/_version.py,sha256=negbvJ46whrHEmNR1Zqf2dRqhwgTzSNi4cQRbm22wck,498
pystare/temporal.py,sha256=c11G3RFoYHyFsW5dQOFAyPNAQMcrvOUARanGJ7EZ2Zw,50958
pystare/core.py,sha256=JNcIdZ-gL-KJSFuJrfRItAltSzx6uSB3HOy8mz82zhU,16410
pystare-0.8.13.dist-info/top_level.txt,sha256=nFEpXWK3JtJgh9bx9enxsVi5_b4iH1bL1X7o8rR-yKc,8
pystare-0.8.13.dist-info/WHEEL,sha256=CzyHWKXay4N1oFds0wxFofK9MEs-L6SZ6gHGZF6-4Co,148
pystare-0.8.13.dist-info/RECORD,,
pystare-0.8.13.dist-info/METADATA,sha256=sgWaJruo90dCNyny-qCK5SdQ6zQ8iXruBvBEa5Ksgwc,3839
